torch/__init__.py

```diff
@@ -2294,8 +2294,7 @@ def compile(
     mode: _Union[str, None] = None,
     options: _Optional[_Dict[str, _Union[str, builtins.int, builtins.bool]]] = None,
     disable: builtins.bool = False,
-) -> _Callable[_InputT, _RetT]:
-    ...
+) -> _Callable[_InputT, _RetT]: ...
```

Comment:
I think these elipses are suppose to be pass statements, I'm unsure if overload will work properly `...` statements, right?

Reply:
In type stub, we should use `...` rather than `pass`.

1. The true implementation is in the following code after `@overload`. You can have an arbitrary body in the overload function. They have no effect because the function is re-defined later.

2. In a empty function, both returns `None`.

```python
def foo(): ...
def bar(): pass

foo()  # -> None
bar()  # -> None
```

3. In `.pyi` stub files, we need to use `...` rather than `pass`. See https://docs.astral.sh/ruff/rules/pass-statement-stub-body. Based on 1. and 2. above, we should also use `...`.